src/Custom-sample.h

```diff
@@ -110,26 +110,26 @@
 #define DEFAULT_PIN_RESET_BUTTON                (-1)
 
 
-#define DEFAULT_USE_RULES                       false             // (true|false) Enable Rules?
+#define DEFAULT_USE_RULES                       true              // (true|false) Enable Rules?
 #define DEFAULT_RULES_OLDENGINE                 true
 
-#define DEFAULT_MQTT_RETAIN                     false             // (true|false) Retain MQTT messages?
+#define DEFAULT_MQTT_RETAIN                     true              // (true|false) Retain MQTT messages?
 #define DEFAULT_MQTT_DELAY                      100               // Time in milliseconds to retain MQTT messages
 #define DEFAULT_MQTT_LWT_TOPIC                  ""                // Default lwt topic
 #define DEFAULT_MQTT_LWT_CONNECT_MESSAGE        "Connected"       // Default lwt message
 #define DEFAULT_MQTT_LWT_DISCONNECT_MESSAGE     "Connection Lost" // Default lwt message
 #define DEFAULT_MQTT_USE_UNITNAME_AS_CLIENTID   0
 
-#define DEFAULT_USE_NTP                         false             // (true|false) Use NTP Server
-#define DEFAULT_NTP_HOST                        ""                // NTP Server Hostname
-#define DEFAULT_TIME_ZONE                       0                 // Time Offset (in minutes)
-#define DEFAULT_USE_DST                         false             // (true|false) Use Daily Time Saving
+#define DEFAULT_USE_NTP                         true              // (true|false) Use NTP Server
+#define DEFAULT_NTP_HOST                        "pool.ntp.org"    // NTP Server Hostname
+#define DEFAULT_TIME_ZONE                       60                // Time Offset (in minutes)
+#define DEFAULT_USE_DST                         true              // (true|false) Use Daily Time Saving
 
 #define DEFAULT_LATITUDE                        0.0f              // Default Latitude  
 #define DEFAULT_LONGITUDE                       0.0f              // Default Longitude
 
-#define DEFAULT_SYSLOG_IP                       ""                // Syslog IP Address
-#define DEFAULT_SYSLOG_LEVEL                    0                 // Syslog Log Level
+#define DEFAULT_SYSLOG_IP                       "192.168.1.104"  // Syslog IP Address
+#define DEFAULT_SYSLOG_LEVEL                    LOG_LEVEL_INFO    // Syslog Log Level
```

Comment:
Do not set this syslog settings in Custom-sample.h, as this will lead to lots of issues if someone doesn't have a syslog server at that IP.